### PR TITLE
removed wrong values added correct values

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #8474 , so i  removed wrong values, added correct values.
i enabled this policy , and i verified on registry.

Capture

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-update.md [hunk 1]
```diff
@@ -1110,8 +1110,8 @@ ADMX Info:
 <!--/ADMXMapped-->
 <!--SupportedValues-->
 Supported values:  
--   true - Enable
--   false - Disable (Default)
+-   0  - Disable
```
Comment (illfated) [inline windows/client-management/mdm/policy-csp-update.md:1113]: The disabled state is the default state, isn't it?
```suggestion
-   0  - Disable (Default)
```

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ManikaDhiman** [comment]
Hi @Kateyanne, Thanks for merging this PR. I was actually awaiting confirmation from the policy owner as there is a discrepancy in the default supported value listed here and the DDF. I just got the confirmation that the update in this PR is correct, hence we are good with the merge. Thanks!

**Comment by Kateyanne** [comment]
Hi @ManikaDhiman - Phew! I'm glad it worked out. Let me talk with @e0i about how this could have gone more smoothly for you, and to prevent in the future. Thanks for your patience.

**Pull request merged**